### DISCUSSION TITLE
Strange E-Step issue

### DISCUSSION THREAD

**Discussion by smooreace** [Jyers/Marlin#884] [Q&A] [answered]
To give a little background, I'm running a v2 with a Microswiss DD and hot-end. Prior to switching to this firmware, I calibrated my e-steps with a final setting of 94. Prints came out great. Super strong and great adhesion. 

Now I'm running this firmware and I'm having what appears to be significant under-extrusion. I verified via printer menu and an M92 that it's actually set to 93 now, which shouldn't be a big problem. 

I ran through the process here - https://teachingtechyt.github.io/calibration.html#esteps

Previous E-Steps - 93
Measured filament at -15.25 (yes, G91, G1 E100 F50 cooked off 135.25mm)

Which gives me an E-Step calc of 68.76

If I didn't have an under extrusion issue before then I sure will with that setting. So that brings me to the question...

If I have, in configuration.h, `#define DEFAULT_AXIS_STEPS_PER_UNIT   { 80, 80, 400, 93 }` is it at all possible that the 93 setting there is stacking with a setting elsewhere?

**Comment by Jyers** [comment]
Certainly sounds strange. There isn't any way the setting is getting stacked or anything as you mentioned. Let me know if you get any more information and I can try to help as best I can.

**Comment by smooreace** [comment, marked as answer]
E-steps were supposed to be set to 136. I'm not sure where the 93 came
from, or why the calculation for the target e-steps was so far off, but it
is resolved now. Set to 136 through the menu and saved settings...
confirmed with an M92

**Reply by scaramonga** [reply]
93 is default for the V2 on stock hotend. Micro Swiss DD needs to be set @ 130 as a starting point (says this in manual), then calibrated from there :)  Mine ended up @ 138.4.

**Reply by smooreace** [reply]
Ya, the strangeness was how it all worked out the first 2 times I calculated things... going from the default 93, to 94. Then testing again and it wanting 68. There is no version of math where that should make sense. Finally ended up at 138.6
